src/defaults.ts

```diff
@@ -1,11 +1,9 @@
 export const networks: { [key: string]: { [key: string]: string } } = {
   ethereum: {
     '1': 'main',
-    '5': 'goerli',
     '11155111': 'sepolia',
     '100': 'xdai',
-    '137': 'matic-main',
-    '80001': 'matic-mumbai'
```

Comment:
Is there any error handling if the user trying to use an unsupported chain? If not should we add??

Reply:
I was just winging it and was trying to get my crypto to a wallet plus I know f all about it tbh